packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -42,52 +62,39 @@ export function DatePicker({
   activator,
   inline,
   selected,
+  readonly = false,
+  disabled = false,
+  fullWidth = false,
 }: DatePickerProps) {
   const datePickerClassNames = classnames(styles.datePicker, {
     [styles.inline]: inline,
   });
+  const wrapperClassName = classnames(styles.datePickerWrapper, {
+    [styles.fullWidth]: fullWidth,
+  });
 
   const datePickerRef = useRef() as RefObject<HTMLDivElement>;
 
   return (
-    <div className={styles.datePickerWrapper} ref={datePickerRef}>
+    <div className={wrapperClassName} ref={datePickerRef}>
       <ReactDatePicker
         calendarClassName={datePickerClassNames}
         showPopperArrow={false}
         selected={selected}
         inline={inline}
+        disabled={disabled}
+        readOnly={readonly}
         onChange={handleChange}
         formatWeekDay={date => date.substr(0, 3)}
         customInput={
-          activator ? (
-            activator
-          ) : (
-            <Button
-              variation="work"
-              type="tertiary"
-              icon="calendar"
-              ariaLabel="Open Datepicker"
-            />
-          )
+          <DatePickerActivator activator={activator} fullWidth={fullWidth} />
         }
         renderCustomHeader={props => <DatePickerCustomHeader {...props} />}
-        onCalendarOpen={focusSelectedDate}
       />
     </div>
   );
 
-  function handleChange(val: Date) {
-    onChange && onChange(val);
-  }
-
-  function focusSelectedDate() {
-    const selectedDateClass = ".react-datepicker__day--selected";
-
-    const selectedDate =
-      datePickerRef.current?.querySelector(selectedDateClass);
-
-    if (selectedDate instanceof HTMLDivElement) {
-      selectedDate.focus();
-    }
+  function handleChange(value: Date) {
+    onChange(value);
   }
```

Comment:
@chris-at-jobber you cool with the date picker not focusing on the selected value when it opens? I think the reason behind this is that it when the input opens, it'll remove the focus from the input. Then a bunch of weird things happens after that

Reply:
I think it's fine to keep the input focused (esp. since the user clicked on an input). If the user `tab` keys their way into the picker they have access, otherwise they can enter the date in the input as the default means of entry

Reply:
@chris-at-jobber also removes it if the activator is just a button

Reply:
What gets removed?

Reply:
@chris-at-jobber oh right. so 
- if you go to https://atlantis.getjobber.com/components/date-picker and click the button, the focus moves to the selected date (assuming you have a selected date)
- if you go to https://deploy-preview-783--jobber-atlantis.netlify.app/components/date-picker and do the same as above bullet, it won't focus on the selected date anymore